test/controllers/update/update.trusts.associated.with.entity.controller.spec.ts

```diff
@@ -90,17 +91,8 @@ describe('Update - Trusts - Trusts associated with the overseas entity', () => {
       expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
     });
 
-    test.each([
-      [
-        "when trust ceased date feature flag off, reviewed trust status will not", false, 0, 0
-      ],
-      [
-        "when trust ceased date feature flag on, reviewed trust status will", true, 1, 2
-      ],
-    ])('%s be shown in summary table', async (_, ceasedDateFeatureFlagValue, expectedRemovedCount, expectedActiveCount) => {
-      mockIsActiveFeature.mockReturnValueOnce(true);
-      mockIsActiveFeature.mockReturnValueOnce(ceasedDateFeatureFlagValue); // FEATURE_FLAG_ENABLE_CEASE_TRUSTS
-
+    test('when trust has a cease date, reviewed trust status will be shown in summary table', async () => {
```

Comment:
this description should just be 'Reviewed trust status will be shown in summary table' as it doesn't depend on the ceased date being provided or not.